ledger/src/blockstore.rs

```diff
@@ -3843,18 +3868,19 @@ fn is_newly_completed_slot(slot_meta: &SlotMeta, backup_slot_meta: &Option<SlotM
             || slot_meta.consumed != backup_slot_meta.as_ref().unwrap().consumed)
 }
 
+// Returns a boolean indicating whether a slot has received additional shreds
+// that can be replayed since the previous update to the slot's SlotMeta.
```

Comment:
nit: Use `///` instead?

Reply:
It's a private but doesn't hurt; needed to rebase anyways so made the change